apollo/client.py

```diff
@@ -139,7 +139,24 @@ def use(cls, provider, token="Beta_token123", *args, **kwargs):
             return f"Provider {provider} not found"
 
     @classmethod
-    def detectText(cls, text, operator, threshold, *args, **kwargs):
+    def detectText(cls, *args, **kwargs):
+        """
```

Comment:
Python documentation can be approached from many different angles. I find the best way to do it is in the following manner: 

"""python
<write a one line description or two line summary of the function> 
:param <param_name>: short 5 word descriptor for arg/param
:type <param_name>: data type (str, dict, list etc..)
:return: Current or empty state is an example or whatever it is 
:rtype: data type
"""

Reply:
Noted. Let me make that change.